code/game/objects/items/weapons/storage/dakis.dm

```diff
@@ -31,20 +32,20 @@
 			desc = "A large pillow depicting [custom_name] in a compromising position. Featuring as many dimensions as you."
 	else
 		if(!spam_flag)
-			spam_flag = 0
-			if(user.a_intent == "help")
+			spam_flag = FALSE
+			if("help")
```

Comment:
But what I meant was we have defines for help, harm, etc. Like `INTENT_HELP` use those instead of raw strings